src/main/java/org/folio/dataexp/service/SingleFileProcessor.java

```diff
@@ -53,24 +53,23 @@ public void exportBySingleFile(UUID jobExecutionId, ExportRequest exportRequest,
     while (exportIterator.hasNext()) {
       var export = exportIterator.next();
       exportRequest.setLastExport(!exportIterator.hasNext());
-      executeExport(export, exportRequest, commonExportFails, !exportIterator.hasNext());
+      executeExport(export, exportRequest, commonExportFails);
       log.info("Export from {} to {} has been executed.", export.getFromId(), export.getToId());
       if (Boolean.TRUE.equals(exportRequest.getAll())) {
         updateStatisticsForExportAll(jobExecutionId);
       }
     }
   }
 
-  public void executeExport(JobExecutionExportFilesEntity export, ExportRequest exportRequest, CommonExportFails commonExportFails,
-      boolean lastExport) {
-    exportExecutor.export(export, exportRequest, commonExportFails, lastExport);
+  public void executeExport(JobExecutionExportFilesEntity export, ExportRequest exportRequest, CommonExportFails commonExportFails) {
+    exportExecutor.export(export, exportRequest, commonExportFails);
   }
 
   private void updateStatisticsForExportAll(UUID jobExecutionId) {
     var jobExecutionEntity = jobExecutionEntityRepository.getReferenceById(jobExecutionId);
     var jobExecution = jobExecutionEntity.getJobExecution();
     var progress = jobExecution.getProgress();
-    progress.setTotal(progress.getExported() + progress.getDuplicatedSrs() + progress.getFailed());
+    progress.setTotal(progress.getExported() - progress.getDuplicatedSrs() + progress.getFailed());
```

Comment:
Hi @obozhko-folio , it is better to get total information by retrieving that data from database side. If something will be missed for export it will easer to understand this.